docs/ai/dotnet-ai-ecosystem.md

```diff
@@ -8,24 +8,62 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
 |---------------|------------------|----------------------|--------------|
 | [Microsoft.SemanticKernel](https://www.nuget.org/packages/Microsoft.SemanticKernel/) | [OpenAI models](https://platform.openai.com/docs/models/overview)<br/>[Azure OpenAI supported models](/azure/ai-services/openai/concepts/models) | [Semantic Kernel](https://github.com/microsoft/semantic-kernel) (Microsoft) | [Semantic Kernel documentation](/semantic-kernel/) |
 | [Azure OpenAI SDK](https://www.nuget.org/packages/Azure.AI.OpenAI/) | [Azure OpenAI supported models](/azure/ai-services/openai/concepts/models) | [Azure SDK for .NET](https://github.com/Azure/azure-sdk-for-net) (Microsoft) | [Azure OpenAI services documentation](/azure/ai-services/openai/) |
+| [OpenAI SDK](https://www.nuget.org/packages/OpenAI/) | [Azure OpenAI supported models](https://platform.openai.com/docs/models) | [OpenAI SDK for .NET](https://github.com/openai/openai-dotnet) (OpenAI) | [OpenAI services documentation](https://platform.openai.com/docs/overview) |
+
+### .NET SDKs for Azure AI Services
+
+Azure offers many other AI services to build specific application capabilities and workflows. Most of these services provide a .NET SDK to integrate their functionality into custom apps. Visit the [Azure AI Services](/azure/ai-services/what-are-ai-services) documentation for a complete list of available services and learning resources. Some of the most commonly used services include the following:
+
+| Service | Description |
+| --- | --- |
+| [Azure AI Search](/azure/search/) | Bring AI-powered cloud search to your mobile and web apps. |
+| [Azure AI Content Safety](/azure/ai-services/content-safety/) | Detect unwanted or offensive content. |
+| [Azure AI Document Intelligence](/azure/ai-services/document-intelligence/) | Turn documents into intelligent data-driven solutions. |
+| [Azure AI Language](/azure/ai-services/language-service/) | Build apps with industry-leading natural language understanding capabilities. |
+| [Azure AI Speech](/azure/ai-services/speech-service/) | Speech to text, text to speech, translation, and speaker recognition. |
+| [Azure AI Translator](/azure/ai-services/translator/) | AI-powered translation technology with support for more than 100 languages and dialects. |
+| [Azure AI Vision](/azure/ai-services/computer-vision/) | Analyze content in images and videos. |
+
+## Develop with local AI models
+
+.NET apps can also connect to local AI models for many different development scenarios. [Semantic Kernel](https://github.com/microsoft/semantic-kernel) is the recommended tool to connect to local models using .NET. Semantic Kernel can connect to many different models hosted across a variety of platforms and abstracts away lower level implementation details.
+
+For example, you can use [Ollama](https://ollama.com/) to run various AI models locally, including several Small Language Models (Slms) developed by Microsoft:
```

Comment:
We should add a link to the Local Models quickstart, to help demonstrate how to use these.